collections/_posts/2023-07-06-Mean_teachers_are_better_role_models.md

```diff
@@ -0,0 +1,203 @@
+
+Mean Teacher also improves results on CIFAR-10 over our baseline $\Pi$ model.
+
+> **NOTE :** The Virtual Adversarial Training (VAT) by [Miyato et al. 2017](https://doi.org/10.48550/arXiv.1704.03976)[2] performs even better than Mean Teacher on the **1000-label SVHN** and the **4000-label CIFAR-10**. VAT and Mean Teacher are complimentary approaches.  
+&rarr; Their combination may yield better accuracy than either of them alone.
+
+## Error percentage over 10 runs on SVHN with extra unlabeled training data :
+
+![](/collections/images/mean_teachers/table_3.jpg)
+
+## Effect of using mean teacher : 
+
+![](/collections/images/mean_teachers/figure_3.jpg)
+
+- the **EMA-weighted models** (blue and dark gray curves in the bottom row) give **more accurate predictions** than the bare student models (orange and light gray) after an initial period. 
+ 
+    &rarr; EMA-weighted model as the teacher **improves results** in the semi-supervised settings
+
+- When using **500 labels** **Mean Teacher** **learns faster**, and continues training after the $\Pi$ model stops improving. 
+
+    &rarr; **Mean Teacher uses unlabeled training data more efficiently than the $\Pi$ model**
+
+- In the **all-labeled case**, Mean Teacher and the $\Pi$ model **behave identically**. With 500k extra unlabeled examples $\Pi$ model keeps improving for longer.
+
+     &rarr; **Mean Teacher learns faster**, and eventually **converges to a better result**, but the sheer amount of data appears to offset $\Pi$ model’s worse predictions.
+
+## Mean Teacher with residual networks on CIFAR-10 and ImageNet 
+
+Checking if the methods scales to more natural images on ImageNet with 10% of the labels : 
+
+![](/collections/images/mean_teachers/table_4.jpg)
+
+# Conclusions
+
+Advantages over Temporal Esembling :
+- **more accurate target labels**, which enables **faster feedback loop** between the student and the teacher model
+- scales to **large datasets** and online learning.
+
+To conclude : 
+- The success of consistency regularization depends on the quality of teacher-generated targets.
+- Mean Teacher and Virtual Adversarial Training represent
+two ways of exploiting this principle.
+
+    &rarr; Their combination may yield even better targets. 
+
+# References
+[1]: Rasmus, Antti, Berglund, Mathias, Honkala, Mikko, Valpola, Harri, and Raiko, Tapani. Semi-supervised Learning with Ladder Networks. In Cortes, C., Lawrence, N. D., Lee, D. D., Sugiyama, M., and Garnett, R. (eds.), Advances in Neural Information Processing Systems 28, pp. 3546–3554. Curran Associates, Inc., 2015  
+
+[2]: Miyato, Takeru, Maeda, Shin-ichi, Koyama, Masanori, and Ishii, Shin. Virtual Adversarial Training: a Regularization Method for Supervised and Semi-supervised Learning. arXiv:1704.03976 [cs, stat], April 2017. arXiv: 1704.03976.
+
+[3] Laine, Samuli and Aila, Timo. Temporal Ensembling for Semi-Supervised Learning. arXiv:1610.02242 [cs], October 2016. arXiv: 1610.02242.
```

Comment:
Another thing I suggest is to shorten your references (e.g. only first author + _et al._, conference/journal + year), but to insert links directly to the papers. That way it's easier for readers to follow up on the references you include, w/o needing to include to much details in the footnotes.